docs/demo/gaiden-css/scss/base.scss

```diff
@@ -6,6 +6,7 @@
 
 body {
   padding: 20px;
+  font-size: 14px;
```

Comment:
@eduardojmatos i set `font-size` to `14px`, because at color list and gradients list demo the caption for each color looks to big for me. 
But we could define this just for `ul.colors` and `ul.gradients`, what do you think?